Assets/Scripts/Audio/TimeStamper.cs

```diff
@@ -0,0 +1,456 @@
+using ProefExamen.Audio.WaveFormDrawer;
+using System.Collections.Generic;
+using UnityEngine;
+using System.Linq;
+using UnityEditor;
```

Comment:
```suggestion
using System.Collections.Generic;
using UnityEngine;
using System.Linq;
using UnityEditor;

using ProefExamen.Audio.WaveFormDrawer;
```
According to code conventions.